pkgs/development/python-modules/websockets_client/default.nix

```diff
@@ -0,0 +1,20 @@
+{ stdenv, buildPythonPackage, fetchPypi,
+  six, backports_ssl_match_hostname, unittest2, argparse }: 
+buildPythonPackage rec {
+  pname = "websocket_client";
+  version = "0.46.0";
+
+  src = fetchPypi {
+    inherit version pname;
+    sha256 = "0f1p9zkzpw0q1cqwnfn788mj3fjqg5z3vsd9pjnz50dk12znngwk";
+  };
+
+  testInputs = [ unittest2 ];
```

Comment:
`checkInputs`? However, it seems like this is not needed.

Reply:
I did reuse the old exprission. I have fixed it and removed `unittest2` from the `propagatedBuildInputs`.

Thanks for spotting it.

Reply:
`unittest2` is only needed for python < 2.7, which isn't supported by Nixpkgs. I'd say let's get rid of it.
```
if sys.version_info[0] == 2 and sys.version_info[1] < 7:
        tests_require.append('unittest2==0.8.0')
```